### PR TITLE
Search form

### PR DESCRIPTION
**Pull request by marinacod**
## Description

Added the Search Form to Home Page. It will let a user search the hotels by different parameters such as destination, checkin date, checkout date, number of guests

## Acceptance Criteria
Search form is responsive

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

### After
image

## Testing Steps / QA Criteria

Put in a destination, check-in date, checkout date, number of guests, number of rooms and click the "Search" button. It should bring you to the next page.

### CODE REVIEW

File: src/api/fetchHotels.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+export default async function fetchHotels(searchData) {
+  const searchParams = new URLSearchParams(searchData);
+  const res = await fetch(
+    `${process.env.REACT_APP_BASE_URL}/booking/hotelsbylocation?${searchParams}`,
+    {
+      method: 'GET',
+
```
Comment (madebymiguel) [inline src/api/fetchHotels.js:7]: Remove space

**Review by madebymiguel — CHANGES_REQUESTED** [review]
Just that small thing

**Review by madebymiguel — APPROVED** [review]
Looks good to me

File: src/api/fetchHotels.js (hunk 1 above)
Comment (OxanaMl) [inline src/api/fetchHotels.js]: Remove empty line

**Review by OxanaMl — CHANGES_REQUESTED** [review]
Everything else looks good

**Review by madebymiguel — APPROVED** [review]

**Review by OxanaMl — APPROVED** [review]

**Review by OxanaMl — APPROVED** [review]

**Pull request merged**